packages/admin-ui/src/lib/static/vendure-ui-config.json

```diff
@@ -23,11 +24,264 @@
         "uk",
         "it",
         "fa",
-        "ne",
-        "hr"
+        "en"
+    ],
+    "availableLocales": [
```

Comment:
Rather than listing all the locales here, I think it would be better to default to an empty array, and adjust the logic in other parts so that if the array is empty, the full list from the shared constant is used. And then only if the developer wants to limit to specific locales, they then need to fill values in this array.

Reply:
done ✅
Actually did that because of non-empty `availableLanguages` array also I need to recheck the default values initialization mechanism and make sure that everything is going to run as expected.

Reply:
I figured out that if we make `availableLanguages` or `availableLocales` empty, the admin-ui angular development mode (http://localhost:4200/admin/) will be broken with the following error message as nothing can be loaded as availableLanguages or availableLocales.

> sample error message
```
Error: The defaultLanguage "en" must be one of the availableLanguages []
```

But if we build the project and access the admin-ui (http://localhost:3000/admin/), there will be a new file created with default values filled and the admin-ui is working as expected.

> static/sample source-control tracked file
```
packages\admin-ui\src\lib\static\vendure-ui-config.json
```

> admin-ui-plugin generated file
```
packages\admin-ui-plugin\lib\admin-ui\vendure-ui-config.json
```

Reply:
so if you are ok with this, we can pre-fill `availableLanguages` & `availableLocales` inside source-control tracked version of `vendure-ui-config.json` to make development mode of admin-ui work as expected and if this is not the desired change let me know and we can find a better solution.